src/document/DocumentAnnotator.scss

```diff
@@ -11,15 +11,41 @@
         pointer-events: none;
     }
 
-    &.ba-is-highlighting {
+    &.ba-discoverability-enabled {
         .textLayer {
-            > span {
-                @include ba-HighlightCreator-cursor;
+            pointer-events: none; // Ensures that click events are received on the create region layer
+
+            span {
+                pointer-events: auto; // Text layer still receives text highlight events
+            }
+        }
+
+        &.ba-is-create--highlight {
+            .textLayer {
+                > span {
+                    @include ba-HighlightCreator-cursor;
+                }
+
+                ::selection {
+                    background-color: rgba($bdl-yellow, .33);
+                }
             }
+        }
```

Comment:
These styles aren't `.ba-discoverability-enabled` specific, right?

Reply:
Yes, confirmed that this is an existing feature.